source/braille.py

```diff
@@ -1569,6 +1574,7 @@ def bufferPosToRegionPos(self, bufferPos):
 		raise LookupError("No such position")
 
 	def regionPosToBufferPos(self, region, pos, allowNearest=False):
+		start: int = 0

```

Comment:
Ig you want proper annotations, please annotate the RegionWithPositions class properly. See https://www.geeksforgeeks.org/typing-namedtuple-improved-namedtuples/

Above `def _get_regionsWithPositions(self):`, add:
```
	#: typing information for autoproperty _get_regionsWithPositions(
	regionsWithPositions: Generator[RegionWithPositions, None, None]
```

Reply:
I suppose there was sometimes "UnboundLocalError" as to start variable. I guess for some reason there was no data in `self.regionsWithPositions` so start might be referenced before assignment. Reverting this.

Reply:
Ugh, now you're saying that, the code in this method indeed assumes that start is set, so I guess it is fine to leave this as is. Good catch.